README.md

```diff
@@ -0,0 +1,34 @@
+# Overview
+
+A tool for dumping pod logs and describing k8s resources owned or created by a juju controller and
+its models.
+
+## Getting Started
+
+This package can be installed via pip and then called as a command line tool
+
+```
+pipx install .
+juju-crashdump-k8s --help
+
+usage: juju-k8s-crashdump [-h] kubeconf controller
```

Comment:
I think weebl-tools uses `cog` to automatically format this, could be a cool addition but isn't necessary.